tsweb/tsweb_test.go

```diff
@@ -502,6 +509,12 @@ func TestStdHandler(t *testing.T) {
 				}
 				return e.Error()
 			})
+			if onStartRecord.Time.IsZero() {
+				t.Error("onStart callback not called")
+			}
```

Comment:
You could compare onStartRecord to test.wantLog but specify the fields only known at completion with cmpopts.IgnoreFields().

Reply:
Done